components/Ink/src/MDCInkView.m

```diff
@@ -193,7 +205,17 @@ - (CGFloat)maxRippleRadius {
 }
 
 - (void)setMaxRippleRadius:(CGFloat)radius {
-  if (self.inkLayer.maxRippleRadius != radius) {
+  _maxRippleRadius = radius;
+  BOOL rippleRadiusChanged = !MDCCGFloatEqual(self.inkLayer.maxRippleRadius, radius);
+  // New Ink will only update inkLayer.maxRippleRadius when using Unbounded ink
+  if (!self.usesLegacyInkRipple) {
+    if (self.inkStyle == MDCInkStyleUnbounded && rippleRadiusChanged) {
```

Comment:
is it more accurate to say here if it isn't Bounded rather than if it's Unbounded? Looks like we want to not update maxRippleRadius only when inkStyle is not Bounded?

Reply:
The style is `.unbounded` so I'm trying to work with that terminology. I'm not sure how negating the value of an `NS_ENUM` would make things clearer. Would you prefer to use a `switch` statement instead?

Do you have a suggestion you could put in a snippet?

Reply:
From what I understand we want the `maxRippleRadius` to change only in the case where it is not bounded right? Because bounded it needs to be set to 0 no matter what. In that case all other options should allow to have the `maxRippleRadius` to be set, and not only .unbounded. As an example if we add more `inkStyle`s in the future

Reply:
I've refactored to use a `switch`. Please take a look.